gen.go

```diff
@@ -552,6 +593,35 @@ func emitCborUnmarshalStructField(w io.Writer, f Field) error {
 	}
 }
 
+var decodeInt = `
+	var extraI int64
```

Comment:
this needs to be scoped, otherwise structs with multiple int64s will bork

Reply:
It is scoped, line 588